OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -292,10 +292,10 @@ void GeometryPath::constructProperties()
  */
 void GeometryPath::namePathPoints(int aStartingIndex)
 {
-    char indx[5];
     for (int i = aStartingIndex; i < get_PathPointSet().getSize(); i++)
     {
-        sprintf(indx,"%d",i+1);
+        char indx[5];
```

Comment:
I understand why you'd think 4 chars (+ nul) is enough here, because no model is going to contain >9999 path points but, for sanity's sake, I'd just use a 32/64/128 buffer here such that the full range of digits can be captured.